Backend/config/config.cfg

```diff
@@ -7,7 +7,8 @@ port = 3306
 database = bauamt
 
 [jwt-secret]
-secret = test
+secret = 496e64697370757461626c792d557273696e652d53746174656d656e742d3030

```

Comment:
Der Hash für die JWTs hat in der Config nix verloren!

Reply:
Fixxed